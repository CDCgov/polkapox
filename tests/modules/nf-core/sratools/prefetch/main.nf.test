nextflow_process {

    name "Test Process SRATOOLS_PREFETCH"
    script "modules/nf-core/sratools/prefetch/main.nf"
    process "SRATOOLS_PREFETCH"

    test("Prefetch Test") {

        when {
            params {
            
                           }
            process {
                """
               input[0] = Channel.of([
                    [ meta: 'SRR29268321'],
                    [ id: 'SRR29268321'  ]
                ])
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
