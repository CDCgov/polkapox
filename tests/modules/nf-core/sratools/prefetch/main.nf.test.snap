{
    "Prefetch Test": {
        "content": [
            {
                "0": [
                    
                ],
                "sra": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-07-09T13:24:29.103330323"
    }
}